resources/js/components/Tool.vue

```diff
@@ -59,25 +55,39 @@
     import ClickToEdit from "./ClickToEdit";
 
     export default {
+        props: {
+            panel: {required: true}
+        },
+
         components: {ClickToEdit},
         data: () => {
             return {
                 apiURL: '/nova-vendor/laravel-nova-translation-editor/',
                 active_locale: Nova.config.locale,
+                locales: [],
                 translations: null,
                 active_tab: null
             }
         },
         mounted() {
-            this.getTranslations();
+            this.fetchMultilingualData();
         },
         methods: {
-            getTranslations() {
-                Nova.request().get(this.apiURL + 'index').then(response => {
+            fetchMultilingualData: function(){
+                this.getTranslations();
```

Comment:
Je doet dit nog steeds in 2 aparte calls, kan dit niet in 1 call? Of is dat niet handig?